src/status_im/ui/screens/wallet/choose_recipient/views.cljs

```diff
@@ -81,11 +81,12 @@
     [react/view {:style styles/wallet-container}
      [status-bar/status-bar {:type :wallet}]
      [toolbar-view camera-flashlight]
-     [react/view {:style     styles/qr-container
-                  :on-layout #(let [layout (.. % -nativeEvent -layout)]
-                                (re-frame/dispatch [:set-in [:wallet/send-transaction :camera-dimensions]
-                                                    {:width  (.-width layout)
-                                                     :height (.-height layout)}]))}
+            [react/view {:style         styles/qr-container
+                         :pointerEvents :none
```

Comment:
Is this whole logic something we want to package for reuse?

Reply:
It's tricky, because in most cases the camera does have buttons or something that use touch-events, so this fix won't fork for the other cameras we use, for example. Ultimately there's an issue in the `react-native-camera` library so this is a bit of a hack